Directory.Build.props

```diff
@@ -26,6 +26,9 @@
     <NonNet6EssentialsPlatforms Condition="'$(BuildNonNet6ForAndroid)' == 'true'">netstandard2.0;netstandard2.1;MonoAndroid10.0</NonNet6EssentialsPlatforms>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
 
+    <!-- HACK: WinUI seems to have issues without this -->
+    <GenerateLibraryLayout>true</GenerateLibraryLayout>
```

Comment:
This is needed because there seems to be an issue when using the generated .xbf files in class libraries.